src/App.js

```diff
@@ -1,18 +1,17 @@
-import React, {useState} from 'react';
-import TodoList from './TodoList';
-import AddTodoForm from './AddTodoForm';
-import {LIST_TITLES} from './data/data';
-import {buildList} from './utils/listUtils';
+import React, { useState } from "react";
+import TodoList from "./TodoList";
+import AddTodoForm from "./AddTodoForm";
 
 const App = () => {
-    const [newTodo, setNewTodo] = useState('');
+    const [todoList, setTodoList] = useState([]);
+    const addTodo = (newTodo) => setTodoList(todoList => [...todoList, newTodo]);
```

Comment:
Nitpick (doesn't require changes):
Excellent usage of the state setter callback function! However, while this works as intended, there's some variable confusion between the `todoList` coming from `useState` and the `todoList` variable coming from the `setTodoList` callback function parameter. I'd recommend changing the variable naming within the callback function to create a visual separation of which variable is being used here:
```js
const addTodo = (newTodo) => setTodoList(prevTodoList => ([...prevTodoList, newTodo]));
```